src/canopy-react-error-boundary.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import {CprButton} from 'canopy-styleguide!sofe';
+
+export default function decorateOptions(opts) {
+  if (typeof opts !== 'object' || typeof opts.featureName !== 'string') {
+    throw new Error("canopy-react-error-boundary decorator should be called with an opts object that has a featureName string. e.g. @ErrorBoundary({featureName: 'life'})");
+  }
+
+  return function decorateComponent(Comp) {
+    return class CanopyReactErrorBoundary extends React.Component {
+      state = {
+        caughtError: null,
+        caughtErrorInfo: null,
+        dismissed: false,
+      }
+      render() {
+        if (!this.state.caughtError) {
+          return <Comp {...this.props} caughtError={this.state.caughtError} caughtErrorInfo={this.state.caughtErrorInfo} />
```

Comment:
Do you need to pass these down? It seems like they will be null most of the time, and the `Comp` doesn't really care about them, right?

Reply:
Good point -- this was before i was doing the modal. Will remove.